tests/datatypes/test_bool.py

```diff
@@ -32,6 +32,13 @@ class UnaryBoolOperationTests(UnaryOperationTestCase, TranspileTestCase):
 class BinaryBoolOperationTests(BinaryOperationTestCase, TranspileTestCase):
     data_type = 'bool'
 
+    def test_xor_int(self):
+        self.assertCodeExecution("""
+            x = True
+            x ^ 2
+            #expect 3
+            """)
+
```

Comment:
This extra test is not necessary; the test for xor_int is automatically generated, the `not_implemented` line that you delete below just tell the test suite that the test is now expected to pass.